src/extension.ts

```diff
@@ -134,6 +134,15 @@ export function activate(context: ExtensionContext) {
         runTextInTerm(text);
     }
 
+    async function runFromLineToEnd() {
+        const startLine = window.activeTextEditor.selection.start.line;
+        const startPos = new Position(startLine, 0);
+        const endLine = window.activeTextEditor.document.lineCount;
+        const range = new Range(startPos, new Position(endLine, 0));
```

Comment:
The `0` in `const range = new Range(startPos, new Position(endLine, 0));` should be the last character of the `endLine`? But it works?

Reply:
Lines in VS Code are 0-indexed, so `endLine` is the line _after_ the last line of the file. I reckon that's why it works.